src/profile-logic/tracks.js

```diff
@@ -204,6 +205,25 @@ export function computeGlobalTracks(profile: Profile): GlobalTrack[] {
         globalTracksByPid.set(pid, globalTrack);
       }
     }
+
+    // Check for screenshots.
+    const ids: Set<string> = new Set();
+    if (stringTable.hasString('CompositorScreenshot')) {
+      const screenshotNameIndex = stringTable.indexForString(
+        'CompositorScreenshot'
+      );
```

Comment:
I have a pretty big preference to keeping with the `MarkerTable` practice, rather than a secondary data structure, especially given the follow-up work with the `MarkerChart` processing.

As for the string, I'm not sure on the concern for having a hard coded string, as we key off of many values from the marker table using a string.